### PR TITLE
Geosearch refactor round 0

### PR DESCRIPTION
**Pull request by DonQueso89**
* Use SQL Composition for point queries (this still needs some work but it requires refactoring the way dynamic parts of the query are stored on the class, which id rather do in a separate  PR. Consider this an explorative refactor)
* Move configuration from the code to the environment as much as possible (and reasonable).
* Simplify local development by loading all dbs into a single container
* Remove Telus related stuff
* Update a test assertion because the online data changed (verified this manually against dump). Replacing this with fixtures will be a follow-up PR.

### CODE REVIEW

File: web/geosearch/datapunt_geosearch/datasource.py [hunk 1]
```diff
@@ -131,41 +132,54 @@ def execute_queries(self, datasets=None):
...
-            """.format(
-                self.fields, self.meta["geofield"], table, self.meta["geofield"], self.extra_where
+            stmt = sql.SQL("""
+                SELECT {fields},
+                    ST_Distance({geo_field},
+                    ST_GeomFromText('POINT({x} {y})', 28992)) AS distance
+                FROM {schema}.{table_name}
+                WHERE ST_DWithin({geo_field}, ST_GeomFromText('POINT({x} {y})', 28992), {radius}) {extra_where} {temp_predicate} ORDER BY distance {limit}
+            """).format(
+                fields=sql.SQL(", ").join(map(sql.SQL, self.fields.split(","))), # TODO: Properly escape SQL in self.fields string
```
Comment (lbam) [inline web/geosearch/datapunt_geosearch/datasource.py:170]: Would this work?

```suggestion
                fields=sql.SQL(", ").join(map(sql.Identifier, self.fields.split(","))),
```

File: web/geosearch/datapunt_geosearch/datasource.py [hunk 2]
```diff
@@ -131,41 +132,54 @@ def execute_queries(self, datasets=None):
...
+                table_name=sql.Identifier(table.split(".")[1]),
+                schema=sql.Identifier(table.split(".")[0]),
+                geo_field=sql.Identifier(self.meta["geofield"]),
+                x=sql.Placeholder("x"),
+                y=sql.Placeholder("y"),
+                radius=sql.Placeholder("radius"),
+                extra_where=sql.SQL(self.extra_where),
+                temp_predicate=tmp_bounds,
+                limit=sql.SQL("LIMIT {}").format(self.limit and sql.Placeholder("limit") or sql.SQL("ALL"))
```
Comment (lbam) [inline web/geosearch/datapunt_geosearch/datasource.py:179]:
```suggestion
                limit=sql.SQL("LIMIT {}").format(sql.Placeholder("limit") if self.limit else sql.SQL("ALL"))
```

**Comment by lbam** [comment]
-151 LoC, w00t!

File: web/geosearch/datapunt_geosearch/config.py [hunk 3]
```diff
@@ -61,66 +23,32 @@ def get_db_settings(db: str, docker_host: str, localport: str) -> Dict[str, str]
...
-                                 docker_default='insecure'),
-        'host': get_variable(db=db, varname='host', docker_default=docker_host,
-                             sa_default='localhost'),
-        'port': get_variable(db=db, varname='port', docker_default='5432',
-                             sa_default=localport),
-        'db': get_variable(db=db, varname='database', docker_default=db)
+        'username': os.environ[f"{db.upper()}_DB_USER_OVERRIDE"],
+        'password': os.environ[f"{db.upper()}_DB_PASSWORD_OVERRIDE"],
+        'host':os.environ[f"{db.upper()}_DB_HOST_OVERRIDE"] ,
+        'port':os.getenv(f"{db.upper()}_DB_PORT_OVERRIDE", "5432"),
```
Comment (lbam) [inline web/geosearch/datapunt_geosearch/config.py:29]:
```suggestion
        'host': os.environ[f"{db.upper()}_DB_HOST_OVERRIDE"] ,
        'port': os.getenv(f"{db.upper()}_DB_PORT_OVERRIDE", "5432"),
```

File: web/geosearch/datapunt_geosearch/config.py [hunk 4]
```diff
@@ -61,66 +23,32 @@ def get_db_settings(db: str, docker_host: str, localport: str) -> Dict[str, str]
...
-    **get_db_settings(db='dataservices',
-                      docker_host='dataservices_db',
-                      localport='5409'))
-
+DSN_BAG = _db_connection_string.format(**get_db_settings('bag_v11'))
+DSN_NAP = _db_connection_string.format(**get_db_settings('nap'))
+DSN_MILIEU = _db_connection_string.format(**get_db_settings('milieuthemas'))
+DSN_MONUMENTEN = _db_connection_string.format(**get_db_settings('monumenten'))
+DSN_VARIOUS_SMALL_DATASETS = _db_connection_string.format(**get_db_settings('various_small_datasets'))
+DSN_DATASERVICES_DATASETS = _db_connection_string.format(**get_db_settings('dataservices'))
```
Comment (lbam) [inline web/geosearch/datapunt_geosearch/config.py:42]:
```suggestion


def _make_conn_str(name: str) -> str:
    return _db_connection_string.format(**get_db_settings('name'))


DSN_BAG = _make_conn_str('bag_v11')
DSN_NAP = _make_conn_str('nap')
DSN_MILIEU = _make_conn_str('milieuthemas')
DSN_MONUMENTEN = _make_conn_str('monumenten')
DSN_VARIOUS_SMALL_DATASETS = _make_conn_str('various_small_datasets')
DSN_DATASERVICES_DATASETS = _make_conn_str('dataservices')
```

(Untested)

File: web/geosearch/datapunt_geosearch/datasource.py (hunk 1 above)
Comment (DonQueso89) [reply web/geosearch/datapunt_geosearch/datasource.py:170]: dit zou ik graag willen maar omdat de waarden in self.fields dus van alles kunnen zijn heb ik die TODO erbij gezet en voor nu ff SQL() instances ervan gemaakt

**Review by lbam — COMMENTED** [review]
LGTM.

**Pull request merged**